### DISCUSSION TITLE
How to add embedding function for existing table

### DISCUSSION THREAD

**Discussion by Raghulkannan14** [lancedb/lancedb#1249] [Q&A]
I have created the table and loaded the embedding manually, now i need to do hybrid search, while doing its throws error,
ValueError: No embedding function for embedding.

I have registered the embedding model like this
embeddings= registry.get("openai").create(name=embed_model)

How to fix this issue.

**Comment by lightsing** [comment]
Same.
How can I add the embedding definition to an existing table and backfill all existing data embeddings?
